pan-os Documentation and metadata improvements (#10029)
demisto/content: Packs/PAN-OS/Integrations/Panorama/README.md

## Use Cases
* Create custom security rules in Palo Alto Networks PAN-OS.
* Creating and updating address objects, address-groups, custom URL categories, URL filtering objects.

> * Use the URL Filtering category information from Palo Alto to enrich URLs by checking the *use_url_filtering* parameter. A proper licnese on the Firewall is needed.

Reviewer: ```suggestion
* Use the URL Filtering category information from Palo Alto Networks to enrich URLs by checking the *use_url_filtering* parameter. A valid license for the Firewall is required.
```
